config/sampleCfg_UL17.cfg

```diff
@@ -0,0 +1,163 @@
+GGF_Graviton280  = /nfs/dust/cms/user/kramerto/hbt_resonant_run2/HHSkims/SKIMS_uhh_2017_v5_16Oct23/SKIM_GGF_Graviton280
+GGF_Graviton300  = /nfs/dust/cms/user/kramerto/hbt_resonant_run2/HHSkims/SKIMS_uhh_2017_v5_16Oct23/SKIM_GGF_Graviton300
+GGF_Graviton320  = /nfs/dust/cms/user/kramerto/hbt_resonant_run2/HHSkims/SKIMS_uhh_2017_v5_16Oct23/SKIM_GGF_Graviton320
+GGF_Graviton350  = /nfs/dust/cms/user/kramerto/hbt_resonant_run2/HHSkims/SKIMS_uhh_2017_v5_16Oct23/SKIM_GGF_Graviton350
+GGF_Graviton400  = /nfs/dust/cms/user/kramerto/hbt_resonant_run2/HHSkims/SKIMS_uhh_2017_v5_16Oct23/SKIM_GGF_Graviton400
+GGF_Graviton450  = /nfs/dust/cms/user/kramerto/hbt_resonant_run2/HHSkims/SKIMS_uhh_2017_v5_16Oct23/SKIM_GGF_Graviton450
+GGF_Graviton500  = /nfs/dust/cms/user/kramerto/hbt_resonant_run2/HHSkims/SKIMS_uhh_2017_v5_16Oct23/SKIM_GGF_Graviton500
+GGF_Graviton550  = /nfs/dust/cms/user/kramerto/hbt_resonant_run2/HHSkims/SKIMS_uhh_2017_v5_16Oct23/SKIM_GGF_Graviton550
+GGF_Graviton600  = /nfs/dust/cms/user/kramerto/hbt_resonant_run2/HHSkims/SKIMS_uhh_2017_v5_16Oct23/SKIM_GGF_Graviton600
+GGF_Graviton650  = /nfs/dust/cms/user/kramerto/hbt_resonant_run2/HHSkims/SKIMS_uhh_2017_v5_16Oct23/SKIM_GGF_Graviton650
+GGF_Graviton700  = /nfs/dust/cms/user/kramerto/hbt_resonant_run2/HHSkims/SKIMS_uhh_2017_v5_16Oct23/SKIM_GGF_Graviton700
+GGF_Graviton750  = /nfs/dust/cms/user/kramerto/hbt_resonant_run2/HHSkims/SKIMS_uhh_2017_v5_16Oct23/SKIM_GGF_Graviton750
+GGF_Graviton800  = /nfs/dust/cms/user/kramerto/hbt_resonant_run2/HHSkims/SKIMS_uhh_2017_v5_16Oct23/SKIM_GGF_Graviton800
+GGF_Graviton850  = /nfs/dust/cms/user/kramerto/hbt_resonant_run2/HHSkims/SKIMS_uhh_2017_v5_16Oct23/SKIM_GGF_Graviton850
+GGF_Graviton900  = /nfs/dust/cms/user/kramerto/hbt_resonant_run2/HHSkims/SKIMS_uhh_2017_v5_16Oct23/SKIM_GGF_Graviton900
+GGF_Graviton1000 = /nfs/dust/cms/user/kramerto/hbt_resonant_run2/HHSkims/SKIMS_uhh_2017_v5_16Oct23/SKIM_GGF_Graviton1000
+GGF_Graviton1250 = /nfs/dust/cms/user/kramerto/hbt_resonant_run2/HHSkims/SKIMS_uhh_2017_v5_16Oct23/SKIM_GGF_Graviton1250
+GGF_Graviton1500 = /nfs/dust/cms/user/kramerto/hbt_resonant_run2/HHSkims/SKIMS_uhh_2017_v5_16Oct23/SKIM_GGF_Graviton1500
+GGF_Graviton1750 = /nfs/dust/cms/user/kramerto/hbt_resonant_run2/HHSkims/SKIMS_uhh_2017_v5_16Oct23/SKIM_GGF_Graviton1750
+GGF_Graviton2000 = /nfs/dust/cms/user/kramerto/hbt_resonant_run2/HHSkims/SKIMS_uhh_2017_v5_16Oct23/SKIM_GGF_Graviton2000
+GGF_Graviton2500 = /nfs/dust/cms/user/kramerto/hbt_resonant_run2/HHSkims/SKIMS_uhh_2017_v5_16Oct23/SKIM_GGF_Graviton2500
+GGF_Graviton3000 = /nfs/dust/cms/user/kramerto/hbt_resonant_run2/HHSkims/SKIMS_uhh_2017_v5_16Oct23/SKIM_GGF_Graviton3000
+
+## specify here if a sample should use a user-defined efficiency bin for the normalization
+## tipical case: HH reweight, every new point needs to be normalised to a different weight sum
+## if nothing is specified, default bin num. 1 is used
+## NOTE: empty sections are ok for this config parser
+
+[userEffBin]
+#HH = 1
```

Comment:
The sample CfgFiles should be modified in this PR only if the same name is required for each sample. If so we should write the desired names in each file. If not each group could upload his one in different PRs with the latest skims once produced.

Reply:
Having the same name simplifies all future merging enormously. I propose doing it now, so we don't have to worry about this ever again. Notice that each group can still have the name of the skims as they want (the paths).

Reply:
For reference, the ```sampleCfg*``` files have been updated.